src/cbc_sdk/platform/asset_groups.py

```diff
@@ -233,6 +254,23 @@ def create_group(cls, cb, name, description, **kwargs):
         group.save()
         return group
 
+    @classmethod
+    def get_all_groups(cls, cb):
+        """
+        Retrieve all asset groups in the organization.
+
+        Required Permissions:
+            group-management(READ)
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+
+        Returns:
+            list[AssetGroup]: List of ``AssetGroup`` objects corresponding to the asset groups in the organization.
+        """
+        return_data = cb.get_object(AssetGroup.urlobject.format(cb.credentials.org_key))
+        return [AssetGroup(cb, v['id'], v, False, False) for v in return_data['results']]
```

Comment:
Would recommend removing false, false as those are defaulted to make it cleaner. I wasn't sure at first what those were

```
[AssetGroup(cb, v['id'], v) for v in return_data['results']]
```

Reply:
OK.